front-end/src/app/transactions/transaction-group-a/transaction-group-a.component.html

```diff
@@ -39,6 +39,7 @@ <h3>Additional Information</h3>
         [form]="form"
         [formSubmitted]="formSubmitted"
         [descriptionIsSystemGenerated]="isDescriptionSystemGenerated(transactionType)"
+        [schedule]="'A'"
```

Comment:
maybe you could set this with `transactionType.scheduleId`

Reply:
:+1: